vit-servicing-station-lib/src/db/queries/snapshot.rs

```diff
@@ -0,0 +1,103 @@
+use crate::{
+    db::{
+        models::snapshot::{Contributor, Snapshot, Voter},
+        schema::{contributors, snapshots, voters},
+        DbConnection, DbConnectionPool,
+    },
+    v0::errors::HandleError,
+};
+use diesel::{ExpressionMethods, Insertable, QueryDsl, RunQueryDsl};
+
+pub async fn query_all_snapshots(pool: &DbConnectionPool) -> Result<Vec<Snapshot>, HandleError> {
```

Comment:
how often is this called? what is the context for its use?

Reply:
It is used in this places https://github.com/input-output-hk/vit-servicing-station/blob/46e15f4f3f3e40aa19720ae7a322781e53bb370d/vit-servicing-station-lib/src/v0/endpoints/snapshot/mod.rs#L85